app/assets/javascripts/application/confetti.js

```diff
@@ -0,0 +1,11 @@
+jQuery(document).on('ready', function() {
+    var flakes = '',
+        randomColor;
+    for(var i = 0, len = 400; i < len; i++) {
```

Comment:
Is the ‘len’ the duration of the confetti? If so, we should be able to make that variable when calling this JavaScript.